Add README for bookmarks and page-redder samples (#1101)
GoogleChrome/chrome-extensions-samples: functional-samples/sample.page-redder/README.md

# Page Redder

> This sample demonstrates how to use [action API](https://developer.chrome.com/docs/extensions/reference/api/action) to execute code when the icon extension is clicked.

Reviewer: ```suggestion
This sample demonstrates how to use [action API](https://developer.chrome.com/docs/extensions/reference/api/action) to execute code when the extension icon is clicked.
```

> This sample demonstrates how to use [action API](https://developer.chrome.com/docs/extensions/reference/api/action) to execute code when the extension icon is clicked.

Reviewer 2: ```suggestion
This sample demonstrates how to use the [Action API](https://developer.chrome.com/docs/extensions/reference/api/action) to execute code when the extension icon is clicked.
```

> This sample demonstrates how to use the [Action API](https://developer.chrome.com/docs/extensions/reference/api/action) to execute code when the extension icon is clicked.

Reviewer: Similar to above.

```suggestion
This sample demonstrates how to use the [chrome.action](https://developer.chrome.com/docs/extensions/reference/api/action) API to execute code when the extension icon is clicked.
```

## Overview

> The extension change the background color of the active tab page (if its URL doesn't start with `chrome://`) to red when the icon extension is clicked.

Reviewer: ```suggestion
This extension changes the background color of the active tab page (if its URL doesn't start with `chrome://`) to red when the extension icon is clicked.
```

## Running this extension

1. Clone this repository.
2. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
3. Navigate to any page (make sure that the URL doesn't start with `chrome://`).

> 4. Click on the icon extension.

Reviewer: ```suggestion
4. Click on the extension icon.
```
